bzt/modules/apiritif/generator.py

```diff
@@ -409,14 +500,19 @@ def _gen_screenshot_mngr(self, selector):
                 args=[ast.Name(id="filename")]))
         return elements
 
-    def _gen_wait_sleep_mngr(self, atype, tag, param, selector):
+    def _gen_wait_sleep_mngr(self, atype, tag, param, selectors):
         elements = []
         mode = "visibility" if param == 'visible' else 'presence'
 
         if atype == 'wait':
             exc = TaurusConfigError("wait action requires timeout in scenario: \n%s" % self.scenario)
             timeout = dehumanize_time(self.scenario.get("timeout", exc))
-            errmsg = "Element %r failed to appear within %ss" % (selector, timeout)
+            locator_type = list(selectors[0].keys())[0]
+            locator_value = selectors[0][locator_type]
+            errmsg = "Element %r:%r failed to appear within %ss" % (locator_type, locator_value, timeout)
+
+            elements.append(self._gen_get_locators("var_loc_wait", selectors)),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expression "(elements.append(self._gen_get_locators('var_loc_wait', selectors)), )" is assigned to nothing](https://app.codacy.com/app/Taurus/taurus/pullRequest?prid=4568861)